tools/pylint-extensions/azure-pylint-guidelines-checker/pylint_guidelines_checker.py

```diff
@@ -1319,25 +1368,14 @@ def check_parameters(self, node):
             arg_names.append(vararg_name)
 
         docparams = {}
+        keyword_args = {}
         for idx, line in enumerate(docstring):
-            # this param has its type on a separate line
-            if line.startswith("param") and line.count(" ") == 1:
-                param = line.split("param ")[1]
-                docparams[param] = None
-            # this param has its type on the same line
-            if line.startswith("param") and line.count(" ") == 2:
-                _, param_type, param = line.split(" ")
-                docparams[param] = param_type
-            # if the param has its type on the same line with additional spaces
-            if line.startswith("param") and line.count(" ") > 2:
-                param = line.split(" ")[-1]
-                param_type = ("").join(line.split(" ")[1:-1])
-                docparams[param] = param_type
-            if line.startswith("type"):
-                param = line.split("type ")[1]
-                if param in docparams:
-                    docparams[param] = docstring[idx+1]
+            # check for keyword args in docstring
+            keyword_args.update(self._find_keyword(line))
```

Comment:
nit: can we rename this to like docstring_keyword_args? I keep getting confused which collection is holding the docstring kws and which is holding the method kws

Reply:
yes! 

docstring_keyword_args
method_keyword_args